components/cd-alert/cd-alert.css

```diff
@@ -1,31 +1,90 @@
 .cd-alert {
-  padding: 0.5rem 1rem;
+  --cd-flow-space: 0.5rem;
+  padding: 1rem;
   margin: 1rem 0 2rem;
+  border-radius: 0.25rem;
+  border: 1px solid var(--cd-blue--bright);
+  box-shadow: -8px 0 0 var(--cd-blue--bright);
+  color: var(--cd-grey--dark);
+}
+
+.cd-alert,
+/* This is Drupal specific. If it's an error message, it gets an extra div with a role. */
+.cd-alert div[role='alert'] {
```

Comment:
I wonder if we should just include the wrapper unconditionally and set the `role` based on the type of alert.

It would allow us to set one rule for flex container, and avoid implementation errors in the non-drupal websites where markup is also being replicated (e.g. for HID I didn't know this is the case and our system can't even cater to this at present)

Reply:
Yep, good point. I'm adding some feedback from Andy too
https://github.com/UN-OCHA/common_design/pull/141

```
/* If it's a warning or an error message, div should have role='alert' which is equivalent to aria-live='assertive'. */
.cd-alert div[role='alert'],
/* If it's a status message, div should have aria-live='polite' */
.cd-alert div[aria-live='polite'] {
```